lib/rules/jsx-sort-props.js

```diff
@@ -241,6 +261,10 @@ module.exports = {
         shorthandLast: {
           type: 'boolean',
         },
+        // Whether multiline properties should be listed first or last
+        multiline: {
+          enum: ['ignore', 'first', 'last'],
```

Comment:
```suggestion
          enum: ['ignore', 'first', 'last'],
          default: 'ignore',
```
we still need the explicit default, but this is nice documentation